src/cfnlint/custom_rules.txt

```diff
@@ -0,0 +1,3 @@
+#Comments can be made with a '#' symbol at the start of the line
+#Syntax: <Resource Type> <Property[*]> <Operator> <Value>
+#Example: AWS::EC2::Instance InstanceType EQUALS "p3.2xlarge"
```

Comment:
I think I'd prefer different operator syntax
```suggestion
#Example: AWS::EC2::Instance InstanceType == "p3.2xlarge"
```
so just starting a thread here to discuss

Reply:
I'm open to using either types of operators - My only concern in the context of consistency is if in the future there may be operators which are difficult to express with just symbols?

That said, the program has been designed with a single point of reference to make it very straightforward to change in the future, or even include both types of syntax?
`Operator = {'EQUALS': lambda x, y: cfnlint.customRules.Operators.equalsOp(x, y)}`